src/main/resources/archetype-resources/src/main/java/__applicationName__.java

```diff
@@ -11,10 +11,10 @@
 
 @SpringBootApplication
 @Slf4j
-public class TestSpringApplication {
+public class ${applicationName} {
```

Comment:
If I'm not mistaken, validation regex (^[a-zA-Z0-9]+$) accepts for example 6 but would cause compile error